samples/meta-space-warp-sample/main.tscn

```diff
@@ -0,0 +1,184 @@
+tracker = &"left_hand"
+pose = &"grip"
+
+[node name="LeftControllerFbRenderModel" type="OpenXRFbRenderModel" parent="XROrigin3D/LeftHand"]
+
+[node name="Label3D" type="Label3D" parent="XROrigin3D/LeftHand"]
+transform = Transform3D(1, 0, 0, 0, -4.37114e-08, 1, 0, -1, -4.37114e-08, 0, -0.124, -0.186)
+pixel_size = 0.001
+text = "[Analog Stick] Move"
+
+[node name="RightHand" type="XRController3D" parent="XROrigin3D"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0.478861, 0.468292, -0.241097)
+tracker = &"right_hand"
+pose = &"grip"
+
+[node name="RightControllerFbRenderModel" type="OpenXRFbRenderModel" parent="XROrigin3D/RightHand"]
+render_model_type = 1
+
+[node name="Label3D" type="Label3D" parent="XROrigin3D/RightHand"]
+transform = Transform3D(1, 0, 0, 0, -4.37114e-08, 1, 0, -1, -4.37114e-08, 0, -0.124, -0.186)
+pixel_size = 0.001
+text = "Space Warp: ENABLED
+[A] Enable/Disable Space Warp
+[Analog Stick] Turn"
+
+[node name="TurnTimer" type="Timer" parent="XROrigin3D"]
+wait_time = 0.2
+one_shot = true
+
+[node name="Floor" type="MeshInstance3D" parent="."]
+mesh = SubResource("PlaneMesh_mjcgt")
+
+[node name="FloorCollider" type="Area3D" parent="Floor"]
+
+[node name="CollisionShape3D" type="CollisionShape3D" parent="Floor/FloorCollider"]
+shape = SubResource("BoxShape3D_lquwl")
+
+[node name="GPUParticles3D" type="GPUParticles3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -1, 1, -4)
+amount = 2
+process_material = SubResource("ParticleProcessMaterial_yvvw4")
+draw_pass_1 = SubResource("BoxMesh_t5jnj")
+
+[node name="Label3D" type="Label3D" parent="GPUParticles3D"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -1, 0, 0)
+text = "GPU
+Particles"
+
+[node name="CPUParticles3D" type="CPUParticles3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -1, 2.21267, -4)
+amount = 2
+mesh = SubResource("BoxMesh_0xm2m")
+spread = 0.0
+gravity = Vector3(0, 0, 0)
+initial_velocity_min = 3.0
+initial_velocity_max = 3.0
+
+[node name="Label3D" type="Label3D" parent="CPUParticles3D"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -1, 0, 0)
+text = "CPU
+Particles"
+
+[node name="MultiMeshInstance3D" type="MultiMeshInstance3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 3.40488, -4)
+multimesh = SubResource("MultiMesh_1bvp3")
+script = ExtResource("3_h2yge")
+
+[node name="Label3D" type="Label3D" parent="MultiMeshInstance3D"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -2, 0, 0)
+text = "MultiMesh"
+
+[node name="BoneCube" parent="." instance=ExtResource("4_1bvp3")]
+transform = Transform3D(-4.37114e-08, 0, 1, 0, 1, 0, -1, 0, -4.37114e-08, -3, 1, -1)
+
+[node name="Label3D" type="Label3D" parent="BoneCube"]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -1.52612, 0, 0)
+text = "Skeleton"
+
+[node name="MovingCube" type="Node3D" parent="."]
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, -3, 2.399, -1)
+script = ExtResource("5_lquwl")
```

Comment:
This is `moving_cube.gd`. For some reason, it's attached to both the parent `MovingCube` (which is a `Node3D`) and the child `MovingCube` (which is a `MeshInstance3D`)?

This causes the sample not to run for me. But if I detach the script from the parent here, then it runs fine.

Reply:
Huh I wonder why this causes it to not run for you, it runs just fine with both attached. But I've fixed this!